src/cmd/index.js

```diff
@@ -0,0 +1,4 @@
+/* @flow */
+import build from './build';
+import run from './run';
+export {build, run};
```

Comment:
This is probably a good candidate for an `export default {build, run}'.

if we do this kind of change, we need to change the corresponding imports [1] to something like:

```
import defaultCommands from  './cmd';
```

[1]: in https://github.com/mozilla/web-ext/pull/105/files#diff-9589e3a0cc72e26a4814bc9ef05aa2d4R6 
  and 
in https://github.com/mozilla/web-ext/pull/105/files#diff-2d9bd4ae7220a18dfa5b6b6707f45213R7